assets/src/app/containers/containers.component.html

```diff
@@ -5,7 +5,7 @@ <h1>
 <mat-table #table *ngIf="containersPage" [dataSource]="containersPage.data">
   <ng-container matColumnDef="id">
     <mat-header-cell *matHeaderCellDef>ID</mat-header-cell>
-    <mat-cell *matCellDef="let container">{{container.id}}</mat-cell>
+    <mat-cell *matCellDef="let container">{{container.id | short_id}}</mat-cell>
```

Comment:
can be done with predefined slice pipe:
https://angular.io/api/common/SlicePipe
`{{container.id | slice:0:12}}`